frontend/packages/dev-console/integration-tests/views/serverless.view.ts

```diff
@@ -0,0 +1,22 @@
+import { by, element } from 'protractor';
+
+export const firstKebabMenu = element.all(by.css('[data-test-id="kebab-button"]')).first();
+export const actionItems = element(by.css('[data-test-id="action-items"]'));
+export const editCount = element.all(
+  by.cssContainingText('.pf-c-dropdown__menu-item', 'Edit Count'),
+);
+export const machineSetsInput = element(
+  by.className('pf-c-form-control co-m-number-spinner__input'),
+);
+export const submitCount = element(by.id('confirm-action'));
+export const cellClass = element.all(by.css('[data-test-id="openshift-machine-api"]')).first();
+export const pageSidebar = element(by.id('page-sidebar'));
+export const installServerlessOperator = element(
+  by.css('div.pf-c-modal-box.co-catalog-page__overlay.co-catalog-page__overlay--right'),
+);
+export const createProject = element(by.id('yaml-create'));
+export const nameofProject = element(by.id('input-name'));
+export const displayNameofProject = element(by.id('input-display-name'));
+export const servicesValidation = element(by.id('yaml-create'));
+export const serverlessNavItemValidation = element(by.css('[data-test-id="resource-title"]'));
```

Comment:
instead of validating all `revisions/services/routes` against same id 

can we have more specific check i.e along with `resource-title` check for span text as well `Services`

`<span data-test-id="resource-title" class="co-resource-item__resource-name">Services</span>`

Reply:
I will try to add this validation.